src/bundle/Resources/public/js/scripts/admin.location.view.js

```diff
@@ -19,12 +19,35 @@
         },
     };
     const handleEditItem = (content) => {
-        doc.querySelector('#form_subitems_content_edit_content_info').value = content._id;
-        doc.querySelector('#form_subitems_content_edit_version_info_content_info').value = content._id;
-        doc.querySelector('#form_subitems_content_edit_version_info_version_no').value = content.CurrentVersion.Version.VersionInfo.versionNo;
-        doc.querySelector(`#form_subitems_content_edit_language_${content.mainLanguageCode}`).checked = true;
-
-        doc.querySelector('#form_subitems_content_edit_create').click();
+        const contentId = content._id;
+        const checkVersionDraftLink = window.Routing.generate('ezplatform.version_draft.has_no_conflict', { contentId });
+        const submitVersionEditForm = () => {
+            doc.querySelector('#form_subitems_content_edit_content_info').value = contentId;
+            doc.querySelector('#form_subitems_content_edit_version_info_content_info').value = contentId;
+            doc.querySelector('#form_subitems_content_edit_version_info_version_no').value = content.CurrentVersion.Version.VersionInfo.versionNo;
+            doc.querySelector(`#form_subitems_content_edit_language_${content.mainLanguageCode}`).checked = true;
+            doc.querySelector('#form_subitems_content_edit_create').click();
+        };
+        const addDraft = () => {
+            submitVersionEditForm();
+            $('#version-draft-conflict-modal').modal('hide');
```

Comment:
This part sould be cached in a variable, like `const modal = $('#version-draft-conflict-modal');` in the higher scope. Then it should be reused in functions like `addDraft`.